pymc3/tests/test_distributions.py

```diff
@@ -1178,7 +1178,9 @@ def test_multidimensional_beta_construction(self):
 
     def test_rice(self):
         self.pymc3_matches_scipy(Rice, Rplus, {'nu': Rplus, 'sd': Rplusbig},
-                                 lambda value, nu, sd: sp.rice.logpdf(value, b=nu, loc=0, scale=sd))
+                                 lambda value, nu, sd: sp.rice.logpdf(value, b=nu / sd, loc=0, scale=sd))
+        self.pymc3_matches_scipy(Rice, Rplus, {'b': Rplus, 'sd': Rplusbig},
```

Comment:
Does this test work before your last commit with the elementwise fixes? If not, we might want to add one.

Reply:
This test is actually part of #3287 and successfully passes. The current PR (#3289) initially broke it when I tried to use elemwise but now passes. If PR #3287 is merged, I'll rebase the current PR so it will be clearer to see the differences.